src/vm/tests/traits.rs

```diff
@@ -0,0 +1,307 @@
+use vm::types::{Value, TypeSignature, QualifiedContractIdentifier, ResponseData, PrincipalData};
+use vm::types::TypeSignature::{IntType, UIntType, BoolType, ListType, BufferType};
+use vm::types::signatures::{ListTypeData};
+use vm::contexts::{OwnedEnvironment,GlobalContext, Environment};
+use vm::execute as vm_execute;
+use vm::errors::{CheckErrors, RuntimeErrorType, Error};
+use vm::analysis::errors::{CheckError};
+use std::convert::TryInto;
+
+use vm::tests::{with_memory_environment, with_marfed_environment, execute, symbols_from_values};
+
+
+#[test]
+fn test_all() {
```

Comment:
Can you make these separate tests, so if any of them break, we'll know which one broke?  It's fine to leave this test as-is, since you're testing against both a memory and marf'ed environment, but the other methods should also be `#[test]`.